accord-core/src/main/java/accord/local/CommandStores.java

```diff
@@ -181,17 +179,59 @@ public long shards(AbstractKeys<?, ?> keys, long minEpoch, long maxEpoch)
             return accumulate;
         }
 
-        KeyRanges currentRanges()
+        public <T extends Sortable> long shards(Sortables<T, ?> keysOrRanges, long minEpoch, long maxEpoch)
```

Comment:
A lot more `Sortable` is it possible to be more specific like `Routable`?

Reply:
We do operate over keys here for some operations, and I think it is fine in this case to operate over either, logically, without performing a transformation. CommandStores may be partitioned by RoutingKey but the main purpose of Routables is for coordination/inter-node sharing. I am open to reconsidering this, but it felt more reasonable (as well as more efficient) to permit `CommandStores` to operate over either (and to impose stricter requirements further up the API stack, as necessary)

Reply:
Ah, I think the piece I am missing here is that if you use Seekables the seekables are automatically translated to Routable?

Reply:
As discussed offline, we renamed Sortable to Routable, and Routable to Unseeable, to make clear that we could use either here.